Dockerfile.deploy

```diff
@@ -10,10 +10,21 @@ RUN useradd --no-create-home -s /bin/bash ${ALMIGHTY_USER_NAME}
 COPY bin/alm ${ALMIGHTY_INSTALL_PREFIX}/bin/alm
 COPY config.yaml ${ALMIGHTY_INSTALL_PREFIX}/etc/config.yaml
 
+# Install little pcp pmcd server for metrics collection
+# would prefer only pmcd, and not the /bin/pm*tools etc.
+COPY pcp.repo /etc/yum.repos.d/pcp.repo
+RUN yum install -y pcp && yum clean all
+COPY ./alm+pmcd.sh /alm+pmcd.sh
+RUN chmod a+x      /alm+pmcd.sh
```

Comment:
if you change the privileges of the file `alm+pmcd.sh` you don't need this layer which 1) increases the size of the image; 2) add one image layer so one more step to get the docker build.